fre/pp/checkoutScript.py

```diff
@@ -38,6 +51,12 @@ def _checkoutTemplate(experiment, platform, target, branch='main'):
     preexist_error = f"fatal: destination path '{name}' exists and is not an empty directory."
     click.echo(clonecmd)
     cloneproc = subprocess.run(clonecmd, shell=True, check=False, stdout=PIPE, stderr=STDOUT)
+
+    if branch == 'main':
+        local_version = subprocess.run(['fre' ,'--version'])
+        if local_version != '2024.01':
+            subprocess.run(['git' ,'checkout' ,'tags/2024.01'])
```

Comment:
the more and more i think about it, the less this makes sense to me. a few problems are coming to mind

1) this is SHOULD work locally, with an editable pip install of `fre-cli`, if one is in the base directory near the relevant `.git` folder

if a `conda install noaa-gfdl::fre-cli` is being used, this won't do anything! 
2) the first issue is that one's `cwd` may very well not be near the instance of where `fre-cli` actually is... or, consider a user who does `module load fre/canopy`...

3) lets say, somehow, `git checkout tags/2024.01` part works, one would have to `conda install` or re-`pip install` (if not an editable install) into the local environment, but i'm pretty sure one cannot call `pip` programmatically in `python` at all for security reasons.

Reply:
this idea is NOT dead-on-arrival! chris/avery to hash out